setup/generate_usptream_keylime_code_coverage/test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+# define RUST_IMA_EMULATOR variable to install also rust IMA emulator
+
+rlJournalStart
+
+    rlPhaseStartSetup "Build and install rust-keylime bits"
```

Comment:
either the phase name is confusing or you are doing something different than advertized